templates/partial/_footer.html

```diff
@@ -4,7 +4,7 @@
     </div>
     <div class="row">
       <div class="col-9">
-        <p>&copy; {{ current_year }} Canonical Ltd.</p>
+        <p>&copy; 2019 Canonical Ltd.</p>
```

Comment:
I think that comment was about the footer in general, I don't think he meant for this to be changed to a hard coded year. It looks like he left that comment in 2019, this part of the doc probs just hasn't been updated since then

Reply:
We had this discussion before, and apparently the right thing is to keep the [original fixed date it was published](https://stackoverflow.com/a/2391555/2454059). If what he meant is to keep the current year, I think it wouldn't make sense for him to leave that comment, because that's what we've always done.
But I can ask for confirmation

Reply:
It does look like more our sites have conformed with one single format, so I've done the change. But I do remember having done a PR where we were advised to do something else.